docs/turbo/generators/utils.js

```diff
@@ -0,0 +1,51 @@
+const axios = require("axios");
```

Comment:
can we stop using axios for new code and just use native fetch or undici? 😢  (we need to upgrade to node 18 too, so maybe undici till that happens)

Reply:
Yea fair we do need to rip it all out soon